webapp/packages/plugin-sql-editor/src/SqlEditor/SqlEditor/SqlEditor.tsx

```diff
@@ -0,0 +1,84 @@
+/*
+ * CloudBeaver - Cloud Database Manager
+ * Copyright (C) 2020-2024 DBeaver Corp and others
+ *
+ * Licensed under the Apache License, Version 2.0.
+ * you may not use this file except in compliance with the License.
+ */
+import { observer } from 'mobx-react-lite';
+import { useEffect, useMemo, useState } from 'react';
+
+import { getComputed, s, SContext, StyleRegistry, useS, useSplit } from '@cloudbeaver/core-blocks';
+import { useService } from '@cloudbeaver/core-di';
+import { baseTabStyles, ITabData, TabList, tabListStyles, TabPanelList, TabsState, verticalRotatedTabStyles } from '@cloudbeaver/core-ui';
+import { MetadataMap } from '@cloudbeaver/core-utils';
+import { useCaptureViewContext } from '@cloudbeaver/core-view';
+
+import { ISqlEditorModeProps, SqlEditorModeService } from '../../SqlEditorModeService';
+import { DATA_CONTEXT_SQL_EDITOR_DATA } from '../DATA_CONTEXT_SQL_EDITOR_DATA';
+import type { ISqlEditorProps } from '../ISqlEditorProps';
+import { SQLEditorActions } from '../SQLEditorActions';
+import { useSqlEditor } from '../useSqlEditor';
+import styles from './styles/SqlEditor.m.css';
+import tabStyles from './styles/SqlEditorTab.m.css';
+import tabListModuleStyles from './styles/SqlEditorTabList.m.css';
+
+const sqlEditorRegistry: StyleRegistry = [
+  [tabListStyles, { mode: 'append', styles: [verticalRotatedTabStyles, tabListModuleStyles] }],
+  [baseTabStyles, { mode: 'append', styles: [verticalRotatedTabStyles, tabStyles] }],
+];
+
+export const SqlEditor = observer<ISqlEditorProps>(function SqlEditor({ state, className }) {
```

Comment:
This file was moved, but GitHub detected it as a delete + new file. Please fix this. It's very important to save the commit history

Reply:
done